tests/parallel-variants.test.js

```diff
@@ -67,12 +67,12 @@ test('parallel variants can be generated using a function that returns parallel
       .test\:font-bold *::test {
         font-weight: 700;
       }
-      .test\:font-medium *::test {
-        font-weight: 500;
-      }
       .test\:font-bold::test {
         font-weight: 700;
       }
+      .test\:font-medium *::test {
+        font-weight: 500;
+      }
       .test\:font-medium::test {
```

Comment:
I am not sure why, but I had to move the order of 2 classes in 2 separate tests.